docs/notebook-demos/ingesting-an-image.ipynb

```diff
@@ -0,0 +1,198 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Ingesting an Image\n",
+    "\n",
+    "This example shows how to ingest a grayscale image and save the results locally. It is assumed that you have already read through the guides before beginning this tutorial."
```

Comment:
link to which guide.

perhaps not for all workbooks, but for some of them, it will be useful to imagine "what if a possible user was sent a link to this...and needs more background" 

In this case linking to the guides you are presuming theyve read will be enough to help them find their way.